x/profiles/types/expected_keeper.go

```diff
@@ -6,21 +6,10 @@ import (
 	connectiontypes "github.com/cosmos/ibc-go/v3/modules/core/03-connection/types"
 	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
 	ibcexported "github.com/cosmos/ibc-go/v3/modules/core/exported"
-
-	subspacestypes "github.com/desmos-labs/desmos/v4/x/subspaces/types"
 )
 
 // DONTCOVER
 
-// SubspacesKeeper represents the expected keeper used to interact with subspaces
-type SubspacesKeeper interface {
-	// HasSubspace tells if the subspace with the given id exists
-	HasSubspace(ctx sdk.Context, subspaceID uint64) bool
-
-	// GetAllSubspaces returns all the subspaces stored
-	GetAllSubspaces(ctx sdk.Context) []subspacestypes.Subspace
-}
```

Comment:
`x/profiles` is not required to have `x/subspaces` now. It remains since `x/profiles` used to include `x/relationships` which couple with `x/subspaces`.

Reply:
Perfect, thanks for the explanation